src/parcels/_core/utils/sgrid.py

```diff
@@ -378,3 +392,95 @@ def parse_sgrid(ds: xr.Dataset):
         xgcm_coords[axis] = {"center": dim_dim_padding.dim2, xgcm_position: dim_dim_padding.dim1}
 
     return (ds, {"coords": xgcm_coords})
+
+
+def rename_dims(ds: xr.Dataset, dims_dict: dict[str, str]) -> xr.Dataset:
```

Comment:
I thought this naming is nice its in line with the xarray API, and when we use this function it would be

```python
from parcels._core.utils import sgrid
...
sgrid.rename_dims(ds, ...)
```